lib/awesome_spawn.rb

```diff
@@ -120,4 +143,10 @@ def parse_command_options(command, options)
 
     [env, build_command_line(command, params), options]
   end
+
+  def detach(env, command, options)
+    pid = Kernel.spawn(env, command, options)
```

Comment:
I think you can still just use `launch`, and get the pid from `status.pid` for detaching.

Reply:
Would prefer not to use capture for this detach